### PR TITLE
Merge 2.8 into develop

### PR DESCRIPTION
**Pull request by manadart**
Merge from 2.8 to bring forward:
- #11838 from wallyworld/fstab-uuids
- #11833 from manadart/2.8-link-layer-address-extract
- #11821 from wallyworld/azure-bootstrap-fixes
- #11832 from manadart/2.8-interface-infos-dedup
- #11830 from ycliuhw/fix/lp-1887440
- #11825 from wallyworld/merge-2.7-20200713
- #11828 from achilleasa/2.8-make-manual-cleanup-script-more-robust
- #11826 from SimonRichardson/remove-nbd-integration-tests
- #11824 from hpidcock/fix-operator-imagepath
- #11798 from jujubot/increment-to-2.8.2

### CODE REVIEW

**Comment by manadart** [comment]
[Merge 2.8 into develop](https://trello.com/c/Mh0VojuP/2790-merge-28-into-develop)

File: cmd/juju/commands/bootstrap.go [hunk 1]
```diff
@@ -562,6 +574,38 @@ to create a new model to deploy k8s workloads.
...
+				if len(allRegions) == 1 {
+					plural = " is"
+				}
+				ctx.Infof("Available cloud region%v %v", plural, strings.Join(allRegions, ", "))
+			}
+			return errors.NotValidf("region %q for cloud %q", c.Region, c.Cloud)
+		}
+	}
+
+	isCAASController = jujucloud.CloudIsCAAS(cloud)
```
Comment (wallyworld) [inline cmd/juju/commands/bootstrap.go:598]: this could be moved closer to line 649 where it's used

Comment (manadart) [reply cmd/juju/commands/bootstrap.go]: Done.

**Comment by hpidcock** [comment]
$$merge$$

**Pull request merged**